neuromation/api/__init__.py

```diff
@@ -165,10 +165,24 @@ async def login_headless(
     *,
     url: URL = DEFAULT_API_URL,
     path: Optional[Path] = None,
-    timeout: aiohttp.ClientTimeout = DEFAULT_TIMEOUT
+    timeout: aiohttp.ClientTimeout = DEFAULT_TIMEOUT,
 ) -> None:
     await Factory(path).login_headless(get_auth_code_cb, url=url, timeout=timeout)
 
 
 async def logout(*, path: Optional[Path] = None) -> None:
     await Factory(path).logout()
+
+
+class NoProjectRoot(Exception):
+    pass
+
+
+def find_project_root() -> Path:
```

Comment:
Please reuse this function in `load_user_config()` to reduce the code duplication.
Another reason is the future change for search criteria. Perhaps we want to support `.neuro` folder with toml file inside as an option to keep the single space for configs. `neuro-flow` keeps its own scenarios in this way. 
The `.neuro` folder support can wait for a separate PR.